features/step_definitions/cli.rb

```diff
@@ -207,7 +207,7 @@
 # B is the situation when the expected json != the patch json, though the patch succeeds.
 # E.g. if the resource has ONLY ONE label and we remove it, the "labels" key will be removed together, so
 # the expected json should be {"metadata":{"labels":null}}.
-Given /^(?:(as admin) )?I successfully patch resource "(.*)\/(.*)" with:$/ do |as_admin, resource_type, resource_name, table|
+Given /^(?:(as admin) )?I successfully patch resource "(.*)\/(.*)" (?:in #{QUOTED} type )?with:$/ do |as_admin, resource_type, resource_name, type, table|
```

Comment:
I would suggest to change this to:
```
Given /^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$/ do |as_admin, patch_type, resource_type, resource_name, table|
```

Note that you will have to add `OPT_WORD = "(?: #{WORD})?"` at line 9 of `features/step_definitions/_constants.rb`.